CONTRIBUTING.md

```diff
@@ -4,6 +4,8 @@ Thanks for your interest in improving Storybook! We are a community-driven proje
 
 Please review this document to help to streamline the process and save everyone's precious time.
 
+This guide assumes you're using `yarn` as package manager. You can use `npm` as well, but it would be less deterministic, as we only commit `yarn.lock` to the repo.
```

Comment:
I definitely see the concern here and I think yarn is 100% the better package installer. If we're gonna proceed with `yarn` for the package installer of choice, we should make that very clear and keep to it.